src/vs/workbench/services/log/common/keyValueLogProvider.ts

```diff
@@ -53,13 +53,13 @@ export abstract class KeyValueLogProvider extends Disposable implements IFileSys
 				size: 0
 			};
 		}
-		return Promise.reject(new FileSystemError(resource, FileSystemProviderErrorCode.FileNotFound));
+		return Promise.reject(FileSystemError.FileNotFound(resource));
```

Comment:
fyi @sandy081 but also 🙅‍♂ no good usage of API types inside the core. This shouldn't be like that

Reply:
Sorry. Could be a copy/paste issue. Fixed and used proper fs errors.